source/source.go

```diff
@@ -54,7 +56,7 @@ func (s Source) Validate() error {
 
 type SourceBackend interface {
 	String() string
-	Load(ctx context.Context, logger kitlog.Logger) ([]*SourceEntry, error)
+	Load(ctx context.Context, logger kitlog.Logger, client *http.Client) ([]*SourceEntry, error)
```

Comment:
This change to the interface makes it much easier to use a client with a mocked transport.